doc/bin/build-html-doc

```diff
@@ -21,13 +21,20 @@ import re
 import jinja2
 
 
+CHAPTER_PAT = re.compile(r'.+"head-ltitle"\>[-\.\w]+\((?P<chap>\d)\)')
```

Comment:
I suggest:

```python
CHAPTER_PAT_STR = '.+"head-ltitle"\>[-\.\w]+\((?P<chap>\d)\)'
CHAPTER_PAT = re.compile(CHAPTER_PAT_STR)
```

in case you want to display the `CHAPTER_PAT_STR` pattern in logs / stdout or in case you need to change it.

Reply:
Done, thanks